src/validators/bartio/defaultValidatorList.json

```diff
@@ -1,6 +1,14 @@
 {
   "name": "Berachain Bartio Validator List",
   "validators": [
+    {
+      "id": "0x99fC13a5b46491D84494165FFaa540fFE7AB78D1",
+      "logoURI": "https://raw.githubusercontent.com/Contribution-DAO/contributiondao/main/brand-assets/cdao_logo_128x128.png",
```

Comment:
Can you switch this to reference the ContributionDAO logo we have in the assets folder? We automatically upload to cloudinary.

`https://res.cloudinary.com/duv0g402y/raw/upload/src/assets/ContributionDAO.png`

Reply:
Hi @corduroybera Updated 

Thank you